dashboard/api.py

```diff
@@ -350,6 +284,7 @@ def format_courserun_for_dashboard(course_run, status_for_user, certificate=None
         status_for_user (str): a string representing the status of a course for the user
         certificate (Certificate): an object representing the
             certificate of the user for this run
+        position (int): The position of the course run within the list
```

Comment:
It's to tell the dashboard what order the runs are in within a course. Though having them in a list in that order anyway seems to make that unnecessary. We don't actually use it anywhere